src/uwtools/api/jedi.py

```diff
@@ -0,0 +1,54 @@
+"""
+API access to the ``uwtools`` ``jedi`` driver.
+"""
+
+import datetime as dt
+from pathlib import Path
+from typing import Dict, Optional
+
+import uwtools.drivers.support as _support
+from uwtools.drivers.jedi import JEDI as _JEDI
+
+
+def execute(
+    task: str,
+    config: Path,
```

Comment:
What is the reason why `config` is not an `Optional[Path]` here?

Reply:
If #460 merges before this PR, I think this should become `Optional[Union[Path, str]] = None`, and a call made to `_ensure_data_source()` on the value -- see that PR. `graph_file` also needs to support a `str` value; there, too, see that PR. If this merges first, I'll take care of it on my branch for that PR.